realsense2_camera/src/rs_node_setup.cpp

```diff
@@ -258,7 +260,7 @@ void BaseRealSenseNode::startPublishers(const std::vector<stream_profile>& profi
             _info_publishers[sip] = _node.create_publisher<sensor_msgs::msg::CameraInfo>(camera_info.str(),
                                     rclcpp::QoS(rclcpp::QoSInitialization::from_rmw(info_qos), info_qos));
 
-            if (_align_depth_filter->is_enabled() && (sip != DEPTH) && sip.second < 2)
+            if (_align_depth_filter->is_enabled() && (sip != DEPTH))// && sip.second < 2)
```

Comment:
Do you prefer to keep this as a comment?
If yes, then I would add some explanation in the comment

Reply:
No, I will remove it.